redash/destinations/hangoutschat.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import requests
+
+from redash.destinations import *
+from redash.utils import json_dumps
+
+
+class HangoutsChat(BaseDestination):
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "url": {
+                    "type": "string",
+                    "title": "Webhook URL (get it from the room settings)"
+                },
+                "icon_url": {
+                    "type": "string",
+                    "title": "Icon URL (32x32 or multiple, png format)"
+                }
+            },
+            "required": ["url"]
+        }
+
+    @classmethod
+    def icon(cls):
+        return 'fa-bolt'
+
+    def notify(self, alert, query, user, new_state, app, host, options):
+        try:
+            if new_state == "triggered":
+                message = "<b><font color=\"#c0392b\">Triggered</font></b>"
+            else:
```

Comment:
Alerts have 3 states: unknown, normal and triggered. You should probably add a `elif` for the unknown case, otherwise it will be treated as normal.

Reply:
OK I based this off the logic in Slack, but I made the enhancement as you suggested.

If triggered - text will be red. If ok - text will be green. Otherwise text will be neutral.

My understanding is "unknown" will happen for example if the alert is unable to determine the state. For example, you're referencing a column name that does not exist or your query returned 0 rows. Instead of just staying "Unknown" - I tried to be a bit more friendly.

Open to suggestions though

![image](https://user-images.githubusercontent.com/12296644/53832576-9fa98300-3f54-11e9-8373-6c66d5a5a8eb.png)

Reply:
This is perfect 👌 Your understanding is correct. The alert also starts in unknown state, but we don't send a message in this case.

And you're right -- we have the same issue with Slack. And while I was bitten by it myself, I was just lazy to fix it until now 😞